src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useState } from "react";
+
+const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const handleSubmit = (event) => {
```

Comment:
When defining a function it's good practice to leave a line above it empty for readability.